packages/components/src/ExampleComponent/__tests__/ExampleComponent.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react"
```

Comment:
What is this structure you've built for this example component? 👀 You might want to check the plop templates and docs :)

Also, the reason why we had a `docs` folder as opposed to having the stories next to their component file is because the distribution structure wasn't great and it was the easiest way to exclude them from being compiled and included in the dist.

If your AIO can give us easier control over this, then we do not necessarily need so many levels of nesting (personally on the train of keeping tests/docs next to their associated files rather than detached; much more self-contained).

Reply:
> easiest way to exclude them from being compiled and included in the dist.

Exactly, I think this structure makes it easier to configure the tooling, rather than maintaining the regex split between the tsconfig, jest and playwright, we just ignore the folders that don't need to be compiled.

Generally I prefer to have everything that isn't compiled nested away so that I can immediately see the files that I can work on. We're also moving to having both `stories.tsx` and `stories.mdx` which adds more clutter.

Happy to move this to a discussion though.